modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustClientCodegen.java

```diff
@@ -414,7 +456,8 @@ public String toVarName(String name) {
 
     @Override
     public String toParamName(String name) {
-        return toVarName(name);
+        // $ref appears to be all uppercase which is contrary to rustfmt practice so lowercase parameters
+        return toVarName(name.toLowerCase(Locale.ROOT));
```

Comment:
This result in change of variable name (e.g. pet_id => petid).

I'll file a PR to fix it by remove https://github.com/OpenAPITools/openapi-generator/pull/12479/files#diff-8daa2cc862ff0a63dd72dac551951b1bb07675e3cc0609a6dea55d2e85492520R440 instead.

Reply:
Filed https://github.com/OpenAPITools/openapi-generator/pull/12820